src/AWS.Distro.OpenTelemetry.AutoInstrumentation/AwsSpanProcessingUtil.cs

```diff
@@ -54,13 +54,18 @@ internal sealed class AwsSpanProcessingUtil
 
     internal static readonly string SqlDialectPattern = "^(?:" + string.Join("|", GetDialectKeywords()) + ")\\b";
 
-    private const string SqlDialectKeywordsJson = "configuration/sql_dialect_keywords.json";
-
     internal static List<string> GetDialectKeywords()
     {
         try
         {
-            using (StreamReader r = new StreamReader(SqlDialectKeywordsJson))
+            string sqlDialectKeywordsJsonFullPath = "configuration/sql_dialect_keywords.json";
+            string otelDotnetAtuoHome = Environment.GetEnvironmentVariable("OTEL_DOTNET_AUTO_HOME") ?? string.Empty;
```

Comment:
nit: variable naming misspelled should otelDotnetAutoHome

Reply:
fixed.